xbmc/guilib/GUIIncludes.h

```diff
@@ -65,4 +69,3 @@ class CGUIIncludes
   std::set<std::string> m_constantAttributes;
   std::set<std::string> m_constantNodes;
 };
-
```

Comment:
Please leave the empty line at the end of the file because a lot of editors will auto-add it back which will result in another change.